cognite/client/data_classes/aggregations.py

```diff
@@ -187,20 +189,37 @@ def dump(self, camel_case: bool = False) -> dict[str, Any]:
         return {"start": self.start, "count": self.count}
 
 
+class Buckets(UserList):
```

Comment:
This essentially becomes a `List[Any]`. Take a look at how we've addressed that in `CogniteResourceList`. We can make our own `UserList` which correctly types things which we can reuse across these classes.